### ISSUE TITLE
Minify removes environment variables

### ISSUE THREAD

**Issue by mateuszpawlik** [ReproNim/neurodocker#452]
I minified an FSL container using the [documentation](https://www.repronim.org/neurodocker/user_guide/minify.html). However, the environment variables, including `PATH` were not set in the new image, after `docker import`. I don't know if this is a result of `docker export` or `docker import`. I tried to find a reason but couldn't. I ended up building a new image with a new Dockerfile based on the `import` output and setting the variables again.

**Comment by kaczmarj** [comment]
yes, i don't think `docker export` preserves environment variables. what you did is the way to go... we should update the documentation with this information..

**Comment by mateuszpawlik** [comment]
I don't remember this issue much anymore. In my opinion, a simple note in the docs would suffice.
